src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -3,140 +3,117 @@ title: Getting informed consent for user research
 permalink: /:path/:basename/
 related_order: 140
 ---
-You must have informed consent from participants before you start any research
-activity with them. And you must then act only within the consent the
-participant has given.
+You must have informed consent from participants before you start any research activity with them. And you must then act only within the consent the participant has given.
 
-This guidance explains how we collect and keep evidence of informed consent at
-dxw. It builds on the general guidance on
-[getting informed consent](https://www.gov.uk/service-manual/user-research/getting-users-consent-for-research)
-in the Service Manual.
+This guidance explains how we collect and keep evidence of informed consent at dxw. It builds on the general guidance on [getting informed consent](https://www.gov.uk/service-manual/user-research/getting-users-consent-for-research) in the GOV.UK Service Manual.
 
-## Creating consent materials
+## Using dxw or client consent materials and processes
+
+On different projects you might use dxw branded consent materials and follow our processes, or use consent materials provided by the client and follow their processes.
+
+There are three basic options:
 
-### Using dxw or client consent materials
+1. Use dxw consent materials and processes.
 
-There are two basic options:
+   In this case, we use our own [templates]
+(https://docs.google.com/document/d/1ilYXwrrkl9s4jMRkJm3uCWAON6bnEeYinKvPd-VOz8k/) to create dxw branded consent materials.
```

Comment:
Just one link issue here, as the link (in brackets) has moved to a new line. This should be resolvable via the markdown editor in GitHub, but let me know if you would like me to fix

Reply:
Thanks. Have fixed that one.